examples/user_guide/03-Applying_Customization.ipynb

```diff
@@ -378,7 +372,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Our plot is now rendered with matplotlib. Note that the `opts.Path` option builder now tab completes *matplotlib* keywords because we activated the matplotlib plotting extension beforehand. Specifically, `linewidth` and `linestyle` don't exist in Bokeh where the corresponding options are called `line_width` and `line_dash` instead.\n",
+    "Our plot is now rendered with Matplotlib, in SVG format (try saving the image to disk to see). Note that the `opts.Path` option builder now tab completes *Matplotlib* keywords because we activated the Matplotlib plotting extension beforehand. Specifically, `linewidth` and `linestyle` don't exist in Bokeh, where the corresponding options are called `line_width` and `line_dash` instead.\n",
```

Comment:
```suggestion
     "Our plot is now rendered with Matplotlib, in SVG format (try right-clicking the image in the web browser and saving it to disk to confirm). Note that the `opts.Path` option builder now tab completes *Matplotlib* keywords because we activated the Matplotlib plotting extension beforehand. Specifically, `linewidth` and `linestyle` don't exist in Bokeh, where the corresponding options are called `line_width` and `line_dash` instead.\n",
```

Reply:
I've removed the spaces in your edits, but be sure to check for those (somehow added by Github).

Reply:
You can simply pull the new commits, reload the notebook and save. That will get rid of the whitespace for you but then you'll need to push another commit fixing the whitespace GitHub mysteriously adds.